packages/components/src/__alpha__/SingleSelect/subcomponents/PopoverContents/PopoverContent.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { type Item, type ItemGroup } from '../../SingleSelect'
+import { List } from '../List'
+import { ListItem } from '../ListItem'
+import { ListSection } from '../ListSection'
+
+type Items = {
+  items: ItemGroup[] | Item[]
+}
+
+export const PopoverContent = ({ items }: Items): React.ReactElement => {
+  return (
+    <List>
+      {items.map((item) => {
+        if ('options' in item) {
```

Comment:
I think remove this for now too so when we get to tackling recursive menus we can do it then, so the initial stories should just be very simple too